grin/test/ExtendedSyntax/ParserSpec.hs

```diff
@@ -163,29 +163,29 @@ spec = do
       let before = [prog|
         grinMain =
           case scrut of
-            13.1415   -> pure ()
-            +14.1415  -> pure ()
-            -14.1415  -> pure ()
-            42        -> pure ()
-            +43       -> pure ()
-            -42       -> pure ()
-            64u       -> pure ()
-            (CNode a1 a2 a3 a4 a5) -> pure ()
-            #default  -> pure ()
-            #True     -> pure ()
-            #False    -> pure ()
+            13.1415@_1   -> pure ()
```

Comment:
This looks a bit odd. Why the variable is after the constant? For me _1@13.1415

Reply:
In that case, the code will become quite unreadable if the variable names are long.

Reply:
Add space around `@`.
I'd like to see the value first.
In lambda2 the syntax is even more restricted, only one operation per line. this means that `pure ()` would go into the next line in lambda2. It improves readability.

Reply:
Added spaces.